sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -90,83 +93,86 @@ export class ChangeFeedResponse<T> {
 
 // @public (undocumented)
 export class ClientContext {
-    constructor(cosmosClientOptions: CosmosClientOptions, globalEndpointManager: GlobalEndpointManager);
+    // Warning: (ae-forgotten-export) The symbol "ClientConfig" needs to be exported by the entry point index.d.ts
+    constructor(cosmosClientOptions: CosmosClientOptions, globalEndpointManager: GlobalEndpointManager, clientConfig: ClientConfig);
     // (undocumented)
-    batch<T>({ body, path, partitionKey, resourceId, options, diagnosticContext, }: {
+    batch<T>({ body, path, partitionKey, resourceId, options, diagnosticNode, }: {
         body: T;
         path: string;
         partitionKey: PartitionKey;
         resourceId: string;
         options?: RequestOptions;
-        diagnosticContext?: CosmosDiagnosticContext;
+        diagnosticNode: DiagnosticNodeInternal;
     }): Promise<Response_2<any>>;
     // (undocumented)
-    bulk<T>({ body, path, partitionKeyRangeId, resourceId, bulkOptions, options, diagnosticContext, }: {
+    bulk<T>({ body, path, partitionKeyRangeId, resourceId, bulkOptions, options, diagnosticNode, }: {
         body: T;
         path: string;
         partitionKeyRangeId: string;
         resourceId: string;
         bulkOptions?: BulkOptions;
         options?: RequestOptions;
-        diagnosticContext?: CosmosDiagnosticContext;
+        diagnosticNode: DiagnosticNodeInternal;
     }): Promise<Response_2<any>>;
     // (undocumented)
     clearSessionToken(path: string): void;
     // (undocumented)
-    create<T, U = T>({ body, path, resourceType, resourceId, options, partitionKey, diagnosticContext, }: {
+    create<T, U = T>({ body, path, resourceType, resourceId, diagnosticNode, options, partitionKey, }: {
```

Comment:
let's add diagnosticNode in the end

Reply:
If you notice, this is the input is a js object, (Notice the surrounding {}), hence order doesn't matter here.